atef/widgets/config.py

```diff
@@ -902,8 +902,10 @@ def handle_locking(self, locked: bool):
             self.delete_button.setEnabled(False)
         else:
             self.desc_edit.setFrameShape(self.desc_edit.StyledPanel)
+            color = self.palette().color(QPalette.ColorRole.AlternateBase)
```

Comment:
[Qt docs](https://doc.qt.io/qt-5/qpalette.html#ColorRole-enum) indicate:

Constant | Value | Description
-- | -- | --
QPalette::Base | 9 | Used  mostly as the background color for text entry widgets, but can also be  used for other painting - such as the background of combobox drop down  lists and toolbar handles. It is usually white or another light color.
QPalette::AlternateBase | 16 | Used as the alternate background color in views with alternating row colors (see QAbstractItemView::setAlternatingRowColors()).

Hmm, what does `Base` look like with your theme? 

`Base` looks correct on macOS:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/5139267/178554818-3c482f11-4544-4f9e-a464-34528450e347.png">
<img width="434" alt="image" src="https://user-images.githubusercontent.com/5139267/178554852-18a00a67-0ea8-444d-82ff-d8a379033644.png">

`AlternateBase` on the other hand is a bit duller:
<img width="434" alt="image" src="https://user-images.githubusercontent.com/5139267/178555160-d8516f8e-c827-40d5-b656-aa9cbacaa775.png">
<img width="435" alt="image" src="https://user-images.githubusercontent.com/5139267/178555137-4bce9c58-5942-4260-94ce-7ddbf6d4f434.png">

A separate bug appears to be that the on-load stylesheet is different in either case - or not applied somehow:
<img width="440" alt="image" src="https://user-images.githubusercontent.com/5139267/178555313-b2c96a00-1b48-4aef-9163-295e56d18e4b.png">

Reply:
![image](https://user-images.githubusercontent.com/19717056/178556726-78308f61-66b2-462a-8dbd-c6284c4c84d2.png)
Base looks good to me!